scripts/tayga.initd

```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+description='Simple, no-fuss NAT64 for Linux'
+pidfile=${pidfile:=/run/${RC_SVCNAME}pid}
+datadir=${datadir:=/var/lib/tayga}
```

Comment:
I'm looking at the existing openrc script for tayga in gentoo: https://github.com/gentoo/gentoo/blob/0f5ec113f82e4b0aa635e6ee1c284bbd99036864/net-proxy/tayga/files/tayga.initd#L10 It uses /var/db/tayga for datadir. 

What happens when datadir is changed? Would tayga lose previous state? Because if so, then I'm not sure how to persist state if a Gentoo user updates to new tayga... perhaps the ebuild could move the directory in post install? Or patch this file? I definitely don't want to break backward compatibility.

That said, /var/db doesn't seem to be a part of the FHS so /var/lib is probably more 'correct' than /var/db....